src/utils/grpc_utils.cc

```diff
@@ -1,12 +1,14 @@
 #include "src/utils/grpc_utils.h"
 
-#include "absl/status/statusor.h"
+#include "absl/status/status.h"
```

Comment:
Maybe you don't this line as status.h is added in grpc_utils.h already.

Reply:
Fixed.